### DISCUSSION TITLE
(Projects Beta) Copy and paste cell values like in a spreadsheet.

### DISCUSSION THREAD

**Discussion by eli-crow** [community/community#9467] [Projects and Issues]
It would be very useful to be able to copy and paste things like assignees, milestones, and labels from row to row, or across multiple rows, like is implied by its spreadsheet form factor.

Projects is cool! Excited to see where it goes!

**Comment by RESLibber** [comment]
I second this and also copy down a row - for instance updating all statuses at the same time.
